### PR TITLE
CUMULUS-3805: Dashboard: display and sort records on Updated timestamp

### PR DESCRIPTION
**Pull request by Nnaga1**
**Summary:** Summary of changes

Addresses [CUMULUS-3085 Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3805)

## Changes

- updated Dashboard pages to display and sort on Updated field instead of createdAt or timestamp

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,11 +10,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 This version of the dashboard requires Cumulus API >= v18.2.0
 
 ### Changed
-
+- **CUMULUS-3805**
```
Comment (etcart) [inline CHANGELOG.md:13]: nit: maybe removal of the duration column should be noted here?

**Review by etcart — APPROVED** [review]
this looks good, one nit, but good to go

**Review by jennyhliu — CHANGES_REQUESTED** [review]
1. The tables (utils/table-config) should access updatedAt field instead of createdAt or timestamp.
2. sort by createdAt or timestamp should be changed to updatedAt

**Review by jennyhliu — COMMENTED** [review]
Update test/components/table/list-view.js
initialSortId="timestamp" to initialSortId="updatedAt"

File: app/src/js/utils/table-config/providers.js [hunk 2]
```diff
@@ -27,9 +27,9 @@ export const tableColumns = [
   },
   {
     Header: 'Last Updated',
```
Comment (jennyhliu) [inline app/src/js/utils/table-config/providers.js:29]: Change to Updated to be consistent with other pages.

**Review by jennyhliu — APPROVED** [review]
Nice work

**Pull request merged**